restructuredtext_lint/cli.py

```diff
@@ -12,23 +12,23 @@
     VERSION = version_file.read().strip()
 
 
-def _main(filepaths, format='text', stream=sys.stdout, encoding=None):
+def _main(filepaths, format='text', stream=sys.stdout, encoding=None, level=0):
     error_dicts = []
     error_occurred = False
 
     for filepath in filepaths:
         # Read and lint the file
-        file_errors = lint_file(filepath, encoding=encoding)
+        file_errors = [error for error in lint_file(filepath, encoding=encoding) if error.level >= level]
 
         if not file_errors:
             if format == 'text':
                 stream.write('INFO File {filepath} is clean.\n'.format(filepath=filepath))
         else:
             error_occurred = True
             if format == 'text':
-                for err in file_errors:
+                for error in file_errors:
                     # e.g. WARNING readme.rst:12 Title underline too short.
-                    stream.write('{err.type} {err.source}:{err.line} {err.message}\n'.format(err=err))
+                    stream.write('{err.type} {err.source}:{err.line} {err.message}\n'.format(err=error))
```

Comment:
I appreciate the consistency fix for `err` to `error` but we should either rename the `kwarg` to `error` or remove this change. I try to keep the `kwargs` in `format` one-to-one with variables to prevent confusion